### PR TITLE
Captcha-UI on attendance events

### PR DESCRIPTION
**Pull request by FluidSense**

### CODE REVIEW

File: src/events/components/DetailView/AttendanceEvent.tsx [hunk 1]
```diff
@@ -55,6 +56,9 @@ const AttendanceEvent = ({ event }: IAttendanceEventProps) => {
   const registrationStart = DateTime.fromISO(event.registration_start);
   const registrationEnd = DateTime.fromISO(event.registration_end);
   const cancellationDeadline = DateTime.fromISO(event.unattend_deadline);
+  const sitekey: string = process.env.RECAPTCHA_PUBLIC_KEY || '';
```
Comment (oleast) [inline src/events/components/DetailView/AttendanceEvent.tsx:59]: This should be defined in the `/src/common/constants` folder like everything else we get from env

File: src/events/components/DetailView/AttendanceEvent.tsx [hunk 2]
```diff
@@ -55,6 +57,8 @@ const AttendanceEvent = ({ event }: IAttendanceEventProps) => {
   const registrationStart = DateTime.fromISO(event.registration_start);
   const registrationEnd = DateTime.fromISO(event.registration_end);
   const cancellationDeadline = DateTime.fromISO(event.unattend_deadline);
+  // @ts-ignore
```
Comment (henrikhorluck) [inline src/events/components/DetailView/AttendanceEvent.tsx:60]: Why is this ignored?

Comment (oleast) [reply src/events/components/DetailView/AttendanceEvent.tsx:60]: Because it is not used yet, but it was left as something fore the others to implement

Comment (FluidSense) [reply src/events/components/DetailView/AttendanceEvent.tsx:60]: Yes, exactly what @oleast said!
I should perhaps have included a comment about that with a TODO: Remove when used.

**Pull request merged**